tests/Entity/TagTest.php

```diff
@@ -32,6 +33,9 @@ public function testSetterAndGetter()
         $this->assertInstanceOf(Tag::class, $sut->setName('foo'));
         $this->assertEquals('foo', $sut->getName());
         $this->assertEquals('foo', (string) $sut);
+
+        $this->assertInstanceOf(Tag::class, $sut->setColor('#fffccc'));
+        $this->assertEquals('#fffccc', $sut->getColor());
```

Comment:
Thanks for the addition!

Wouldn't it make more sense to test the trait just once instead of testing every place where it's used?

See https://phpunit.readthedocs.io/en/9.1/test-doubles.html#mocking-traits-and-abstract-classes

Reply:
I am not really testing that the method works as expected, but I rather want to make sure that the methods exists on the object.